pkg/reconcile/pipeline/context/service_test.go

```diff
@@ -43,8 +47,55 @@ var _ = Describe("Service", func() {
 		Expect(res).To(BeNil())
 	})
 
+	It("should return owned resources", func() {
+		var objs []runtime.Object
+		id := uuid.NewUUID()
+		id2 := uuid.NewUUID()
+		ns := "ns1"
+		u := &unstructured.Unstructured{}
+		u.SetGroupVersionKind(schema.GroupVersionKind{Group: "foo.bar", Version: "v1", Kind: "Foo"})
+		u.SetName("foo")
+		u.SetNamespace(ns)
+		u.SetUID(id)
+
+		objs = append(objs, u)
+
+		var children []interface{}
+
+		for _, gvr := range bindableResourceGVRs {
+			gvk := gvr.GroupVersion().WithKind(strings.Title(gvr.Resource)[:len(gvr.Resource)-1])
```

Comment:
The title form `Configmap` is not equivalent to `ConfigMap`. But I think it's ok for testing. An acceptance test would be better to capture a real-world scenario.

Reply:
good catch - I have modified the test to exclude configmaps and check additional path when `List` returns not found error.